CameraButtonsViewControllerTests.swift

```diff
@@ -0,0 +1,282 @@
+//
+//  CameraButtonsViewControllerTests.swift
+//  GiniVision-Unit-Tests
+//
+//  Created by Enrique del Pozo Gómez on 2/19/19.
+//
+
+import XCTest
+@testable import GiniVision
+
+final class CameraButtonsViewControllerTests: XCTestCase {
+    
+    var cameraButtonsViewController: CameraButtonsViewController!
+    var giniConfiguration: GiniConfiguration!
+    var screenAPICoordinator: GiniScreenAPICoordinator!
+    let visionDelegateMock = GiniVisionDelegateMock()
+    let delegateMock = CameraButtonViewControllerDelegateMock()
+    lazy var imageData: Data = {
+        let image = GiniVisionTestsHelper.loadImage(named: "invoice")
+        let imageData = image.jpegData(compressionQuality: 0.9)!
+        return imageData
+    }()
+    
+    override func setUp() {
+        super.setUp()
+        giniConfiguration = GiniConfiguration.shared
```

Comment:
Why are you using the global configuration in tests? Why not create a custom one?